mkdocs.yml

```diff
@@ -73,6 +73,9 @@ nav:
       - guides/burst_insar_product_guide.md
       - Product Guide: guides/burst_insar_product_guide.md
       - StoryMap: https://storymaps.arcgis.com/stories/191bf1b6962c402086807390b3ce63b0" target="_blank
+    - ARIA S1 GUNW:
```

Comment:
As written, this would make it so there's a dropdown option to see the Product Guide. We could make this like AutoRIFT so you click on the "ARIA S1 GUNW" and it takes you to the product guide without the drop down. 
![Screenshot 2025-02-27 at 2 48 06 PM](https://github.com/user-attachments/assets/123efa80-3c43-40b5-9742-8ba0fe14cdd4)